content/docs/command-reference/remote/modify.md

```diff
@@ -294,6 +294,41 @@ $ dvc remote modify myremote endpointurl \
 For more information on configuring Azure Storage connection strings, visit
 [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string).
 
+**Advanced Azure Authentication**
```

Comment:
is it a common term `Advanced Azure Authentication` or should we use Active Directory in the title- to make it precise and clea?

Reply:
I vote for "Azure Active Directory" which seems to be the way they use it in docs like https://docs.microsoft.com/en-us/azure/app-service/configure-authentication-provider-aad

> Also, maybe link to that or another appropriate doc somewhere. Thanks

Reply:
```suggestion
**[Azure Active Directory](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-whatis)**
```